sharpy/client.py

```diff
@@ -102,11 +102,10 @@ def make_request(self, path, params=None, data=None, method=None):
             cleaned_data = data.copy()
             try:
                 del cleaned_data['subscription[ccCardCode]']
+                ccNum = cleaned_data['subscription[ccNumber]']
+                cleaned_data['subscription[ccNumber]'] = ccNum[-4:]
```

Comment:
Trust me, I'm not considering keeping this band-aid around. I'll keep these in mind next time I run into these cases.